yongshin/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULE BE HERE!
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = 127.0.0.1
```

```suggestion
TYPEORM_HOST = localhost
```
OS에 따라 혹은 경우에 따라 localhost는 127.0.0.1이 아닌 경우도 있기 때문에 localhost라고 수정해주세요~